src/ralph/data_center/models/physical.py

```diff
@@ -214,6 +214,8 @@ def get_free_u(self):
 
         def fill_u_list(objects, height_of_device=lambda obj: 1):
             for obj in objects:
+                if not obj[0]:
```

Comment:
after discussion with @xliiv:
* make this condition more explicit: `if obj[0] == 0 or obj[0] is None`
* add some comment that position is None when objects simply does not have (assigned) position and position 0 is for some accessories (pdu?) with left-right orientation and should not be included in free/filled space.